### ISSUE TITLE
Parameter Problems & Deactivate timeout

### ISSUE THREAD

**Issue by UnixCro** [instaloader/instaloader#1371]
Hello dear instaloader developers, 

for a long time it has been annoying me that `instaloader` keeps stopping to download Instagram files, because I always get the error message `429` that `instaloader` has sent too many requests and I have to wait unnecessarily long before I can use it again.  

So I ask you to show me a way to switch this function off completely, because I can safely take the risk that my account will be blocked because time is more important to me.  And anyway with the face verification, only a change password dialog may come up.  The second thing is that the parameter `—fast-update` constantly causes problems, because whoever breaks off too early during a download or the internet connection fails has NOT always `instaloader` an `iterator.xz` or whatever it is called and that leads to the `instaloader` at  Checking the first downloaded image reports that everything has already been downloaded but is not correct …

I think `instaloader` works wrong with this parameter.  If `instaloader` is at 7/50, it fails but you start it again, `instaloader` only looks at 1/50 and says yes, everything is ok.  You could omit the command entirely. This is the only way to guarantee that ALL pictures & videos will be downloaded.  I don't understand the parameter `—latest-stamps` I couldn't use it because I don't understand the difference between `—fast-update`.  

I hope you can help me with my concern.  I adore `instaloader`.

# UPDATE: 

Meanwhile I understand the parameter `—latest-stamps`, this writes a configuration file in which it logs what was last downloaded.  So does the "fast-update" problem seem to solve or not?  No!  The `—latest-stamps` could also be omitted directly, it is just as if one had not stated anything.  `Instaloader` only reports" File Already exist "as standard as well as without this parameter.  Why can't you develop `—latest-stamps` in such a way that it simply logs everything it has downloaded in order to send as few inquiries as possible to Instagram so that you don't have to fix the timeout problem.

# EXAMPLES

```
# 1 WITHOUT -F/—latest-stamps NEW DOWNLOAD

instaloader --login=username  --stories --highlights --no-captions --no-metadata-json --no-compress-json  whatiwant

Loaded session from /data/data/com.termux/files/home/.config/instaloader/session-username.
Logged in as username.
Stored ID for profile whatiwant.
[1/1] Downloading profile whatiwant
whatiwant/2021-10-23_04-39-37_UTC_profile_pic.jpg
Retrieving posts from profile whatiwant.
[ 1/59] whatiwant/2021-12-31_23-49-31_UTC_1.jpg whatiwant/2021-12-31_23-49-31_UTC_2.jpg
[ 2/59] whatiwant/2021-12-30_16-57-31_UTC_1.jpg

CRTL + Z 

Here instaloader downloads as normal ...
```

```
# 2 WITHOUT -F/—latest-stamps AGAIN DOWNLOAD

instaloader --login=username  --stories --highlights --no-captions --no-metadata-json --no-compress-json  whatiwant
Loaded session from /data/data/com.termux/files/home/.config/instaloader/session-username.
Logged in as username.
[1/1] Downloading profile whatiwant
whatiwant/2021-10-23_04-39-37_UTC_profile_pic.jpg already exists
Retrieving posts from profile whatiwant.
[ 1/59] whatiwant/2021-12-31_23-49-31_UTC_1.jpg exists whatiwant/2021-12-31_23-49-31_UTC_2.jpg exists
[ 2/59] whatiwant/2021-12-30_16-57-31_UTC_1.jpg

CTRL + Z

instaloader says when downloading again that everything exists (causes timeout)
```

```
# 3 WITH -F PARAMETER 
instaloader --login=username  --stories --highlights --no-captions --no-metadata-json --no-compress-json  whatiwant -F
Loaded session from /data/data/com.termux/files/home/.config/instaloader/session-username.
Logged in as username.
[1/1] Downloading profile whatiwant
whatiwant/2021-10-23_04-39-37_UTC_profile_pic.jpg already exists
Retrieving posts from profile whatiwant.
[ 1/59] whatiwant/2021-12-31_23-49-31_UTC_1.jpg exists whatiwant/2021-12-31_23-49-31_UTC_2.jpg exists

NO CTRL + Z

Wait ... Finish? -F does not work as already mentioned
```

```
# 4 WITH --latest-stamps PARAMETER

instaloader --login=username  --stories --highlights --no-captions --no-metadata-json --no-compress-json  whatiwant --latest-stamps

Using latest stamps from /data/data/com.termux/files/home/.config/instaloader/latest-stamps.ini.
Loaded session from /data/data/com.termux/files/home/.config/instaloader/session-username.

Logged in as username.
[1/1] Downloading profile whatiwant
whatiwant/2021-10-23_04-39-37_UTC_profile_pic.jpg already exists
Retrieving posts from profile whatiwant.
[ 1/59] whatiwant/2021-12-31_23-49-31_UTC_1.jpg exists whatiwant/2021-12-31_23-49-31_UTC_2.jpg exists
[ 2/59] whatiwant/2021-12-30_16-57-31_UTC_1.jpg exists

CTRL + Z

LOL What did you bring me now ?! Same #2
```

In a nutshell: 

1. Give the opportunity to bypass timeouts.  

2. Parameter `-F` only checks the first one and thinks it is done.  So it would be better to check the last one.  

3. `—latest-stamps` has no function.  It should be edited in such a way that what is already downloaded is logged in order to start again from where it left off.  

Thank you

**Comment by UnixCro** [comment]
Help wanted

**Comment by UnixCro** [comment]
Hi best Instagram developer. First of all, it is an honor for me to be able to communicate with you.  But I can guarantee that these parameters are actually defective and I hope that if you have time to fix them.

**Comment by aandergr** [comment]
It seems to me that you have made up an entirely different logic, and then wonder why the software solves the observed problem in a different way than you imagine. So you are using a flawed way of reasoning. You claim that the two flags are defective simply because they do something different than you expect. However, that would only be erroneous behavior if your expectation of the behavior matched the documented behavior, which it does not.

> Hi Alexander or best Instagram developer.

Instaloader is in no way affiliated with, authorized, maintained or endorsed by Instagram or any of its affiliates or subsidiaries. This is an independent and unofficial project.

**Comment by UnixCro** [comment]
Ok … Thanks for the answer. You say I'm misapplying the parameters.  But sorry, what exactly did I do wrong?  And what else are these parameters useful for?  In my posts above, they don't seem to work.
